packages/input/__tests__/input.spec.ts

```diff
@@ -284,19 +284,17 @@ describe('Input.vue', () => {
         },
       })
       const _ref = wrapper.vm.$refs.textarea
-      await nextTick()
       const originHeight = _ref.$el.clientHeight
 
       _ref.autosize.minRows = 5
       _ref.autosize.maxRows = 5
 
       _ref.resizeTextarea()
       // Textarea height will change
-      setTimeout(()=>{
+      sleep(200).then(() => {
         const nowHeight = _ref.$el.clientHeight
-        expect(originHeight===nowHeight).toBe(false)
-      },1000)
-
+        expect(originHeight).not.toEqual(nowHeight)
+      })
```

Comment:
Better use await syntax to keep consistency here.

```ts
await sleep(200)
const nowHeight = _ref.$el.clientHeight
expect(originHeight).not.toEqual(nowHeight)
```

Change this will be the last request.

Reply:
In jest dom doesn't render,so clientHeight always zero, i  consider textareaStyle can reflect the change

Reply:
That's totally ok to me. The only thing you need to do is to test it see if it matches the expected results.